app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -1030,36 +1025,40 @@ msgstr "Gwneud cais am gyfrifiad ar wahân"
 msgid "To request a census in a different format or for further help, please <a href='{contact_us_url}'>contact us</a>"
 msgstr "I wneud cais am gyfrifiad mewn fformat gwahanol neu am fwy o gymorth, <a href='{contact_us_url}'>cysylltwch â ni</a>"
 
-#: templates/layouts/_base.html:18
+#: templates/layouts/_base.html:19
 msgid "Previous"
 msgstr "Blaenorol"
 
-#: templates/layouts/_base.html:61
+#: templates/layouts/_base.html:62
 msgid "Tell us whether you accept cookies"
 msgstr "Dywedwch wrthym a ydych chi'n derbyn cwcis"
 
-#: templates/layouts/_base.html:62
+#: templates/layouts/_base.html:63
 msgid "We use <a href='{cookie_settings_url}'>cookies to collect information</a> about how you use census.gov.uk. We use this information to make the website work as well as possible and improve our services."
 msgstr "Rydym ni'n defnyddio <a href='{cookie_settings_url}'>cwcis i gasglu gwybodaeth</a> am y ffordd rydych chi'n defnyddio cyfrifiad.gov.uk. Rydym ni'n defnyddio'r wybodaeth hon i sicrhau bod y wefan yn gweithio cystal â phosibl ac i wella ein gwasanaethau."
 
-#: templates/layouts/_base.html:63
+#: templates/layouts/_base.html:64
 msgid "You’ve accepted all cookies. You can <a href='{cookie_settings_url}'>change your cookie preferences</a> at any time."
 msgstr "Rydych chi wedi derbyn yr holl gwcis Gallwch chi <a href='{cookie_settings_url}'>newid eich dewisiadau o ran cwcis</a> ar unrhyw adeg."
 
-#: templates/layouts/_base.html:64
+#: templates/layouts/_base.html:65
 msgid "Accept all cookies"
 msgstr "Derbyn yr holl gwcis"
 
-#: templates/layouts/_base.html:65
+#: templates/layouts/_base.html:66
 msgid "Set cookie preferences"
 msgstr "Gosod dewisiadau o ran cwcis"
 
-#: templates/layouts/_base.html:66
+#: templates/layouts/_base.html:67
 #: templates/partials/introduction/preview.html:30
 #: templates/partials/summary/collapsible-summary.html:14
 msgid "Hide"
 msgstr "Cuddio"
 
+#: templates/layouts/_base.html:114
+msgid "Skip to main content"
+msgstr "Neidio i’r prif gynnwy"
```

Comment:
I know this string is from crowdin but pretty sure there is an error here, the welsh word for content is `cynnwys` - in this instance it should be the mutated version `gynnwys`.

Reply:
@donnap573 fyi

Reply:
@berroar Is it a difference between saying "content" and "contents"?  We're trying to say 'content'.

gov.wales use: Neidio i'r prif gynnwy
but Bangor University use: Neidio i’r prif gynnwys

We were going with Welsh Gov.

Reply:
I think it's that the word used is gynnwy instead of gynnwys? Is that correct @berroar?

Reply:
Think content and contents are both `cynnwys`, just checked a page on gov.wales as an example:https://gov.wales/content-types, if you toggle the languahe to Welsh you can see they use `cynnwys` too